### ISSUE TITLE
EH Search quotes the whole query

### ISSUE THREAD

**Issue by rwchunk** [happypandax/plugins#30]
**Describe the bug**
Search query has quotes around it, which can prevent results from appearing if the query has more than the title alone.

**To Reproduce**
Unpacked to C:\hpx
setup with postgre, import galleries, enable plugins, start trying to match metadata
most folders are limited in information, most only have the artist/title

**Screenshots**
![image](https://user-images.githubusercontent.com/19255865/62072055-6d4e3a00-b203-11e9-93b0-17408280ba4f.png)
![image](https://user-images.githubusercontent.com/19255865/62072954-50b30180-b205-11e9-901c-5a7c114df837.png)
![image](https://user-images.githubusercontent.com/19255865/62072978-5f011d80-b205-11e9-918e-7d572a379900.png)

**Environment:**
 - OS: Windows 10 1803
 - Firefox 66
 - HPX Version - Client 0.1.1, Server 0.10.2, Database 0.7.0, postgreSQL11

**Comment by rwchunk** [comment]
No, I didn't see that. I checked to be sure and it does come like this by default.
https://github.com/happypandax/plugins/blob/de4c4d5ebdd0c465657e42ed5a0d5d36ecb80b33/plugins/EHentai%20Metadata/main.py#L45

**Comment by twiddli** [comment]
try adding this to your `config.yaml`
```
plugin:
    config:
        ehentai-metadata:
            search_query: {title}
```

**Comment by rwchunk** [comment]
Still happening.
around line 12739
[debug - Copy.log](https://github.com/happypandax/plugins/files/3449514/debug.-.Copy.log)

this message also appears while that was in the config:
![image](https://user-images.githubusercontent.com/19255865/62178975-74af3980-b310-11e9-9535-a7bcd34b077a.png)

**Comment by twiddli** [comment]
Oh right, I mean this:
```
plugin:
    config:
        ehentai-metadata:
            search_query: '{title}'
```

**Comment by rwchunk** [comment]
Still happening.

**Comment by rwchunk** [comment]
[debug.log](https://github.com/happypandax/plugins/files/3578039/debug.log)
more log

**Comment by twiddli** [comment]
this should now be fixed in EHentai Metadata `v1.2.0`
